test/run_test.py

```diff
@@ -510,7 +510,7 @@ def run_test(
             )
         )
         unittest_args.extend(test_module.get_pytest_args())
-        replacement = {"-f": "-x"}
+        replacement = {"-f": "-x", "-dist=loadfile": "--dist=loadfile"}
```

Comment:
Why this change? Can you simply pass `--dist` in test.sh?

Reply:
> Why this change? Can you simply pass `--dist` in test.sh?

@desertfire `--dist` would be processed as one of `run_test.py`'s own argument, if I want to use `--dist`, I'd have to add a `--dist` arg definition in `run_test.py`, which I thought is a little too intrusive. On the other hand, `-dist` with a single `-` would just be treated as an "extra arg".